lib/cli.js

```diff
@@ -29,6 +29,11 @@ class PdfOptions {
         describe: 'enable watch mode',
         type: 'boolean'
       })
+      .addOption('debug', {
```

Comment:
I think `debug` is misleading.
What about `preview` or `open`?

Reply:
I would not know what `open` mean without reading the description, but `preview` sounds fine. Updated it.